Podfile.lock

```diff
@@ -513,10 +513,6 @@ DEPENDENCIES:
   - ZIPFoundation (~> 0.9.8)
 
 SPEC REPOS:
-  https://github.com/wordpress-mobile/cocoapods-specs.git:
-    - WordPressAuthenticator
-    - WordPressShared
-    - WordPressUI
```

Comment:
I think that's due to the glitch I discovered when I worked on the feature. See paaHJt-1SO-p2#what-changes for details.

I think that's a bug in CocoaPods, defaulting to use the CDN as source for _transitive_ dependencies, and only use the declared source order for root dependencies. I yet have to open a GitHub issue about this on CocoaPods' repo.

 - Not sure we can do much about it except reporting and waiting for CP to fix it.
 - One way to workaround this is to manually edit the `Podfile.lock` to switch those back to the `cocoapods-specs.git` entry (then `rm -rf Pods/` for good measure), then re-run `pod install`… not ideal but it seems it worked last time I tried
 - If it doesn't work, in the end that's not that big a deal as long as it finds the podspec and the right version – since the podspecs are the same in both cases anyway – even if it's quite unfortunate.

Reply:
Took a look at this and chatted about it with @AliSoftware, and we'll land it as-is for now pending a response from the CocoaPods folks. Thanks!

Reply:
Issue reported in https://github.com/CocoaPods/CocoaPods/issues/10539